test/JDBC/upgrade/16_2/schedule

```diff
@@ -527,4 +527,10 @@ babel_test_int4_numeric_oper_before_16_3
 babel_test_int8_numeric_oper_before_16_3
 babel_test_int2_numeric_oper_before_16_3
 BABEL_3571-15_5-or-15_6-or-16_1-or-16_2
+test_conv_string_to_date-before-16_3
+test_conv_string_to_datetime
+test_conv_string_to_datetime2-before-16_3
+test_conv_string_to_datetimeoffset-before-16_3
+test_conv_string_to_smalldatetime
+test_conv_string_to_time-before-16_3
```

Comment:
Why have you only added these upto 16.2, 15.6 and 14.11? Why not add these all previous versions?

Reply:
Added in all the base versions